source/_integrations/date.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Date
+description: Instructions on how to set up date entities within Home Assistant.
+ha_category:
+  - Date
+ha_release: '2022.12'
+ha_domain: date
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+The Date integration is built for the controlling and monitoring of dates on devices.
+
+Date entities cannot be implemented manually, but can be provided by other integrations. If you are looking for a way to create a similar entity, please take a look at the [Date/Time helper](/integrations/input_datetime).
```

Comment:
```suggestion
Date entities cannot be created manually, but can be provided by other integrations. If you are looking for a way to create a similar entity, please take a look at the [Date/Time helper](/integrations/input_datetime).
```

I think the meaning here is that users cannot manually create those entities and have to use helpers instead. Then I'd say "create" is clearer here.